awesome/CONTRIBUTING.md

```diff
@@ -70,6 +76,32 @@ Now, you are ready to submit a PR for your changes. Please make sure you have go
 
 TBD
 
+#### STEP 4. Resolve conflicts
+
+If there is no conflict in your pr, you can skip this step.
+Otherwise, you can resolve conflicts according to the following solution.
+
+1. go to github actions page, find your pr and copy the output of `Show database differences` and save it as a `.sql` file.
```

Comment:
PR and GitHub action must be capitalized.

Reply:
done